tests/baselines/reference/commonMissingSemicolons.symbols

```diff
@@ -0,0 +1,93 @@
+=== tests/cases/compiler/commonMissingSemicolons.ts ===

```

Comment:
I’d suggest using `// @noEmit: true` and `// @noTypesAndSymbols: true` since you’re only intending to test the error messages.